Vite migration and master branch embedded scaste fix (#644)
scalacenter/scastie: CONTRIBUTING.md

sbt
> startAll
```

> in separate process start vite dev server `yarn dev` in root directory

Reviewer: ```suggestion
in a separate terminal, start the vite dev server by running `yarn dev` in the root directory of the repository
```

open `http://localhost:8080`

> then run `~client/fastLinkJS` if you work on client part or `~sbtRunner/reStart;server/reStart` if you work on server part. 

Reviewer: ```suggestion
then run `~client/fastLinkJS` if you work on the client part or `~sbtRunner/reStart;server/reStart` if you work on the server part. 
```

you can also open `http://localhost:8080/embed.html` to edit the embedded style

### Testing production build

> start sbt production server:

Reviewer: ```suggestion
Start the sbt production server:
```
